topics/icrc_25_signer_interaction_standard.md

```diff
@@ -69,12 +68,14 @@ The transport channel is not required to provide confidentiality.
 
 ## Sessions
 
-ICRC-25 uses sessions to determine the lifetime of granted permission [scopes](#scopes). Permission scopes (see [`icrc25_request_permissions` message](#icrc25_request_permissions)) are granted for the duration of a single session only. 
+ICRC-25 may use sessions to store user-choices and determine the lifetime of granted permission [scopes](#scopes). Permission scopes (see [`icrc25_request_permissions` message](#icrc25_request_permissions)) are granted for the duration of a single session only. 
 
 A session is established when the first permission request is granted. A session can be revoked by the relying party at any time by sending a [`icrc25_revoke_permissions` message](#icrc25_revoke_permissions) to the signer. The signer can also terminate the session at any time and should offer the user a method to do so.
 
 A session must be terminated automatically after a certain period of inactivity. The session might be extended automatically if the interaction between the relying party and the signer is still _actively_ ongoing when the default session timeout is reached. There must be a maximum session duration (regardless of activity).
 
+Signers that do not maintain sessions (i.e. stateless signers) MUST prompt the user to approve each request individually for each method defined in an ICRC-25 extension.
```

Comment:
Let's see how MUST goes in practice. Imagine a situation with sign_challenge: Why would a user need to approve a simple challenge signature simply to prove they own the account instead of having the signer auto-sign? There doesn't seem to be any reason against that.

Reply:
> There doesn't seem to be any reason against that.

That depends heavily on what the challenge signatures are used for. Let's discuss this in today's session.

Reply:
Removed, as discussed in the last working group.